akka-docs/src/test/scala/docs/persistence/testkit/TestKitExamples.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.persistence.testkit
+
+import akka.actor.typed.ActorSystem
+import akka.persistence.testkit._
+import akka.persistence.testkit.scaladsl.PersistenceTestKit
+import akka.persistence.typed.scaladsl.{ Effect, EventSourcedBehavior }
+import com.typesafe.config.ConfigFactory
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class TestKitExamples {
+
+  //#testkit-typed-usecase
+  class TypedSampleSpec extends AnyWordSpecLike {
+
+    val system: ActorSystem[Cmd] = ActorSystem(
+      EventSourcedBehavior[Cmd, Evt, State](
+        persistenceId = ???,
+        eventHandler = ???,
+        commandHandler = (_, cmd) => Effect.persist(Evt(cmd.data)),
+        emptyState = ???),
+      "name",
+      PersistenceTestKitPlugin.config.withFallback(ConfigFactory.defaultApplication()))
+    val persistenceTestKit = PersistenceTestKit(system)
+
+    "Persistent actor" should {
+
+      "persist all events" in {
+
+        val persistentActor = system
+        val cmd = Cmd("data")
+
+        persistentActor ! cmd
+
+        val expectedPersistedEvent = Evt(cmd.data)
+
+        persistenceTestKit.expectNextPersisted("your-persistence-id", expectedPersistedEvent)
+
```

Comment:
The scala version of the `TestKitExamples.xxx` doesn`t showcase `pTK.clearAll` (as the java does). It'd be great to add it here too.

Reply:
will add